embedded-mongodb/src/main/java/com/playtika/testcontainer/mongodb/EmbeddedMongodbBootstrapConfiguration.java

```diff
@@ -64,16 +68,36 @@ ToxiproxyClientProxy mongodbContainerProxy(ToxiproxyClient toxiproxyClient,
     @Bean(value = BEAN_NAME_EMBEDDED_MONGODB, destroyMethod = "stop")
     public GenericContainer<?> mongodb(ConfigurableEnvironment environment,
                                        MongodbProperties properties,
-                                       MongodbStatusCheck mongodbStatusCheck,
-                                       Optional<Network> network) {
-        GenericContainer<?> mongodb =
-                new GenericContainer<>(ContainerUtils.getDockerImageName(properties))
-                        .withEnv("MONGO_INITDB_ROOT_USERNAME", properties.getUsername())
-                        .withEnv("MONGO_INITDB_ROOT_PASSWORD", properties.getPassword())
-                        .withEnv("MONGO_INITDB_DATABASE", properties.getDatabase())
-                        .withExposedPorts(properties.getPort())
-                        .waitingFor(new LogMessageWaitStrategy().withRegEx(".*mongod startup complete.*"))
```

Comment:
You can see the old check here, I don't think the old MongodbStatusCheck class was used at all.